docs/api/ReactWrapper/isEmptyRender.md

```diff
@@ -0,0 +1,23 @@
+# `.isEmptyRender() => Boolean`
+
+Returns true if the component rendered nothing, i.e., null or false.
```

Comment:
Sorry I didn't see this before. I've updated all references of true, null and false with backticks.